docs/auth/usage.mdx

```diff
@@ -268,28 +268,15 @@ User? user = FirebaseAuth.instance.currentUser;
 
 if (user != null && !user.emailVerified) {
   var actionCodeSettings = ActionCodeSettings(
-    url: 'https://www.example.com/?email=${user.email}',
-    dynamicLinkDomain: "example.page.link",
-    android: {"packageName": "com.example.android", "installApp": true, "minimumVersion": "12"},
-    iOS: {"bundleId": "com.example.ios"},
-    handleCodeInApp: true,
-  );
-
-
-User user = FirebaseAuth.instance.currentUser;
-
-if (!user.emailVerified) {
-
-var actionCodeSettings = ActionCodeSettings(
-        url: 'https://www.example.com/?email=${user.email}',
-        dynamicLinkDomain: "example.page.link",
-        androidPackageName: "com.example.android",
-        androidInstallApp: true,
-        androidMinimumVersion: "12",
-        iOSBundleId: "com.example.ios",
-        handleCodeInApp: true);
-
-await user.sendEmailVerification(actionCodeSettings);
+      url: 'https://www.example.com/?email=${user.email}',
+      dynamicLinkDomain: 'example.page.link',
+      androidPackageName: 'com.example.android',
+      androidInstallApp: true,
+      androidMinimumVersion: '12",
+      iOSBundleId: 'com.example.ios',
+      handleCodeInApp: true);
```

Comment:
```suggestion
      handleCodeInApp: true,
      );
```

Reply:
Thank you for your review.
I applied it.